src/main/scala/jio/package.scala

```diff
@@ -28,30 +21,20 @@ package object jio extends JioFiles {
     }
   }
 
-  implicit class PathOps(val p: Path) extends AnyVal {
-
-    def /(name: String): Path = p resolve name
-
-    def mkdir(bits: Long): Path          = createDirectory(p, asFileAttribute(bitsAsPermissions(bits)))
-    def mkfile(bits: Long): Path         = createFile(p, asFileAttribute(bitsAsPermissions(bits)))
-    def exists                           = Files.exists(p, NOFOLLOW_LINKS)
-    def isFile                           = Files.isRegularFile(p, NOFOLLOW_LINKS)
-    def isDirectory                      = Files.isDirectory(p, NOFOLLOW_LINKS)
-    def isSymbolicLink                   = Files isSymbolicLink p
-    def readSymbolicLink                 = Files readSymbolicLink p
-    def allBytes                         = Files readAllBytes p
-    def list: Vector[Path]               = (Files list p).toVector
-    def filename: String                 = p.getFileName.toString
-    def blockSize: Long                  = 512 // FIXME
-    def blockCount: Long                 = (attributes.size + blockSize - 1) / blockSize
-    def size: Long                       = attributes.size
-    def atime: Long                      = attributes.lastAccessTime.toMillis
-    def mtime: Long                      = attributes.lastModifiedTime.toMillis
+  implicit class FileOps(val f: File) extends Pathish[File] {
```

Comment:
I hear what you are saying. Another reason to get rid of it is that I moved to `Files` for quite a few features that did not behave correctly (latest was `renameTo`)